libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/ViewportDataSourceProcessor.kt

```diff
@@ -129,6 +129,24 @@ internal object ViewportDataSourceProcessor {
         } ?: emptyList()
     }
 
+    fun simplifyCompleteRoutePoints(
+        enabled: Boolean,
+        simplificationFactor: Int,
```

Comment:
check if `simplificationFactor==0`, otherwise it throws `ArithmeticException`(need tests for this)
Also, `simplificationFactor` might be negative, it would work, but the factor shouldn't be negative I think